apps/ensadmin-next/src/components/app-sidebar.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { Activity, DatabaseIcon } from "lucide-react";
+import * as React from "react";
+
+import { NavMain } from "@/components/nav-main";
+
+import { Sidebar, SidebarContent, SidebarHeader, SidebarRail } from "@/components/ui/sidebar";
+import { ConnectionSelector } from "./connections/connection-selector";
+
+const navItems = [
+  {
+    title: "Status",
+    url: "/status",
+    icon: Activity,
+  },
+  {
+    title: "APIs",
+    url: "#",
+    icon: DatabaseIcon,
+    isActive: true,
+    items: [
+      {
+        title: "GraphQL",
+        url: "/gql/ponder",
+      },
+      {
+        title: "Subgraph",
+        url: "/gql/subgraph-compat",
+      },
+      {
+        title: "Ponder",
```

Comment:
```suggestion
        title: "Ponder Client",
```

This isn't a "Ponder" API. Ponder supports any arbitrary APIs and it looks like we'll be making some soon. This is a very specific "Ponder Client" API. It's a very specific API type.